extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -76,12 +76,19 @@ class ForumDumper {
 
 	const FORUM_NAMEPSACES = [
 		NS_WIKIA_FORUM_BOARD,
-		NS_WIKIA_FORUM_BOARD_THREAD
+		NS_WIKIA_FORUM_BOARD_THREAD,
+	];
+
+	const COLUMNS_FOLLOWS = [
+		"follower_id",
+		"mw_thread_id",
+		"timestamp",
 	];
 
 	private $pages = [];
 	private $revisions = [];
 	private $votes = [];
+	private $threadIds = [];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$threadIds" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1068)